src/pages/assets/components/FormSend.tsx

```diff
@@ -281,7 +281,7 @@ const FormSend: React.FC<FormSendProps> = props => {
             <div className="description">{t('send.modal1.description')}</div>
             <div className="item">
               <div className="label">{t('send.modal1.label1')}</div>
-              <div className="address">{`${currentSession.wallet.address}`}</div>
+              <div className="address">{`${currentSession.activeAsset?.address}`}</div>
```

Comment:
```suggestion
              <div className="address">{`${currentSession.activeAsset?.address || currentSession.wallet.address }`}</div>
```
For backward compatibility.